src/relay.js

```diff
@@ -233,6 +233,23 @@ export function sequelizeConnection({name, nodeType, target, orderBy: orderByEnu
           model.sequelize.literal('COUNT(1) OVER()'),
           'full_count'
         ]);
+      } else if (options.limit) {
+        // TODO: scope are not applied to the where clause!
+
+        const countQuery = model
```

Comment:
I'd use cloneDeep and conformOptions,
basically you want to do whatever count() does https://github.com/sequelize/sequelize/blob/master/lib/model.js#L1547